src/codeu/chat/client/core/Context.java

```diff
@@ -18,8 +18,10 @@
 import java.util.ArrayList;
 import java.util.Collection;
 
```

Comment:
nit: remove newline, also order imports alphabetically
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing